src/Microsoft.AspNetCore.Hosting/WebHostBuilder.cs

```diff
@@ -224,7 +225,7 @@ private class WrappedApplicationEnvironment : IApplicationEnvironment
         {
             public WrappedApplicationEnvironment(string applicationBasePath, IApplicationEnvironment env)
             {
-                ApplicationBasePath = applicationBasePath;
+                ApplicationBasePath = Path.GetFullPath(applicationBasePath);
```

Comment:
This resolves relative to CurrentDirectory, which is very arbitrary, no? I thought you were going to resolve relative to the exe path?